README.md

```diff
@@ -23,12 +23,40 @@ cd federal-grant-reporting
 cd single-audit
 pipenv shell
 pipenv install
+```
+
+You will also need to set up a local [PostgreSQL](https://www.postgresql.org) database that matches the configuration in `single-audit/single_audit/settings/development.py`. (Alternately, you can create a local database and user and update the app's configuration to match.)
+
+First, create a new Postgres database from the command line:
+
+```bash
+createdb FGR_LOCAL_DB
```

Comment:
@fureigh Turns out the -U flag is optional; it lets you specify which username to use when connecting to the created DB. https://www.postgresql.org/docs/9.1/app-createdb.html

Reply:
@alexsoble :) I thought that might be the case. Thanks for checking!